controllers/book_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Book
```

Comment:
it's not MVC. You shouldn't use it here. Name this folder just Entities.
And put all entities there.

Your class name should be equal to file name